qa/L0_backend_python/examples/test.sh

```diff
@@ -240,6 +240,36 @@ if [ "$TEST_JETSON" == "0" ]; then
     wait $SERVER_PID
 fi
 
+# Example 4
```

Comment:
Instead of Example number just call it "Auto Complete Model Configuration Example"